website/source/docs/configuration/storage/google-cloud.html.md

```diff
@@ -42,6 +45,27 @@ storage "gcs" {
 - `max_parallel` `(string: "128")` – Specifies the maximum number of concurrent
   requests.
 
+- `ha_enabled` `(bool: false)` – Specifies whether this backend should be used
+  to run Vault in high availability mode. This can also be provided via the
+  environment variable `GCS_HA_ENABLED`.
+
+This backend also supports the following high availability parameters. These are
+discussed in more detail in the [HA concepts page](/docs/concepts/ha.html)
+
+- `cluster_addr` `(string: "")` – Specifies the address to advertise to other
```

Comment:
We shouldn't list these here (as they'll quickly become outdated. Just keep the link to the HA page instead.